spec/unit/fibered_mysql2_adapter_spec.rb

```diff
@@ -28,19 +28,25 @@
   context '#lease' do
     subject { adapter.lease }
 
-    it { should eq(Fiber.current) }
+    it { in_concurrent_environment { should eq(Fiber.current) } }
 
     if Rails::VERSION::MAJOR > 4
```

Comment:
You mean 6?

Reply:
Yeah, those tests are unnecessary now, so I got rid of them: https://github.com/Invoca/fibered_mysql2/pull/9/commits/2d2f46502cba52d179255670ba6dfd68d25de034